src/AppInstallerCLICore/Workflows/DependenciesFlow.cpp

```diff
@@ -249,9 +249,9 @@ namespace AppInstaller::CLI::Workflow
         const auto& rootInstaller = context.Get<Execution::Data::Installer>();
         const auto& rootDependencies = rootInstaller->Dependencies;
 
-        if (rootDependencies.Empty())
+        if (rootDependencies.Empty() || context.Args.Contains(Execution::Args::Type::SkipDependencies))

```

Comment:
In theory there should only be one entry for dependencies in the workflow. But I reviewed dependency related workflows and they are scattered, I think we need a refactor to fix the issue. For now, I'm ok adding this block here. But can you move it to the top of this method?